views/navbar.hbs

```diff
@@ -0,0 +1,25 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" href="/">Logo</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="#">About<span class="sr-only">(current)</span></a>
```

Comment:
Why does this have "current"?

Reply:
could "current" be set dynamically based on the current webpage?

Reply:
Took <span class="sr-only">(current)</span> out. the sr-only means screen reader which is used for people who are visually impaired.  It makes it easier for them.